code/datums/components/storage/storage.dm

```diff
@@ -780,7 +780,7 @@
 	return hide_from(target)
 
 /datum/component/storage/proc/on_alt_click(datum/source, mob/user)
-	if(!isliving(user) || !user.CanReach(parent))
+	if(!isliving(user) || !user.CanReach(parent) || user.incapacitated())
 		return
 	if(locked)
 		to_chat(user, "<span class='warning'>[parent] seems to be locked!</span>")

```

Comment:
Remove the now redundant check at the bottom of the proc

Reply:
Wait what do you want me to do? 

If I remove the user.incapacitated() from wallet.dm, but leave the user.incapacitated() in the parent proc in storage.dm the bug still persists. If I remove the user.incapacitated() from storage.dm the bug is still fixed, but it doesn't hurt to leave it in storage.dm, does it?